lib/honeycomb-rails/config.rb

```diff
@@ -48,6 +49,19 @@ def record_flash?
     # The Honeycomb write key for your team (must be specified).
     attr_accessor :writekey
 
+    # If set, determines how to record the sample rate for a given Honeycomb
+    # event. (default: 1, do not sample)
+    #
+    # Valid values:
+    # * Integer - sample Honeycomb events at a constant rate
+    # * 1 or lower - disable sampling on this dataset; capture all events
+    # * TODO: :rails - default Rails dynamic sampling?
+    #
+    # You can also pass a Proc, which will be called with the current controller
+    # instance during each request, and which should return a sample rate for
+    # the request in question.
+    attr_accessor :sample_rate
```

Comment:
Belated: it looks like this comment is out of sync with the implementation (the Proc takes the ActiveSupport::Notifications payload, not the current controller instance). Cowboy fix: 9cdb2c8